docker/docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
         command: --regtest
         ports:
             - '127.0.0.1:4444:4444'
+            - '127.0.0.1:4445:4445'
```

Comment:
This change is already included in an open PR: https://github.com/rsksmart/rif-relay/pull/158/files#diff-423deb13b7c401b1a7f41ee91c77f722e11d2f317d6a66b546524e8a04cc8b03R15